docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.theme

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Theme functions for su_humsci_gin_admin theme.
+ */
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Template\Attribute;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Implements hook_library_info_alter().
+ */
+function su_humsci_gin_admin_library_info_alter(&$libraries, $extension) {
+  $libraries_unset = [
+    'humsci_traditional' => 'humsci-colorful-styling',
+    'humsci_colorful' => 'humsci-traditional-styling',
+  ];
+  if ($extension === 'su_humsci_gin_admin') {
+    $theme = \Drupal::config('system.theme')->get('default');
+    unset($libraries[$libraries_unset[$theme]]);
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function su_humsci_gin_admin_preprocess(&$variables, $hook) {
+  if (!(str_starts_with($hook, 'pattern_') || $hook == 'paragraph')) {
```

Comment:
Is there a hook called `paragraph` or is that supposed to be another str_starts_with condition?

Reply:
There's a `paragraph` hook used for all bundles, it works like the `node` hook.